### PR TITLE
Applied new functional

### PR DESCRIPTION
**Pull request by GlebTheProgrammer**

### CODE REVIEW

File: Application/Services/ActionNotesService.cs [hunk 1]
```diff
@@ -21,6 +21,7 @@ public async Task<List<ActionNote>> GetCustomerNotesWithLimitConnectedWithAccoun
             result.ToList().Sort((x, y) => DateTime.Compare(x.ActionMadeTime, y.ActionMadeTime));
 
             return result.Reverse().Skip(skip).Take(limit).ToList();
+            //return new List<ActionNote>();
```
Comment (VadzimPapko) [inline Application/Services/ActionNotesService.cs:24]: do we need it?

File: Application/Services/ActionNotesService.cs [hunk 2]
```diff
@@ -31,6 +32,7 @@ public async Task<List<ActionNote>> GetCustomerNotesWithLimitConnectedWithOrders
...
```
Comment (VadzimPapko) [inline Application/Services/ActionNotesService.cs:35]: same

File: Application/Services/ActionNotesService.cs [hunk 3]
```diff
@@ -41,6 +43,7 @@ public async Task<List<ActionNote>> GetCustomerNotesWithLimitConnectedWithVehicl
...
```
Comment (VadzimPapko) [inline Application/Services/ActionNotesService.cs:46]: same

File: CarSharingApp.PublicApi/CarSharingApp.PublicApi.csproj [hunk 4]
```diff
@@ -12,6 +12,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Seq.Extensions.Logging" Version="6.1.0" />
```
Comment (VadzimPapko) [inline CarSharingApp.PublicApi/CarSharingApp.PublicApi.csproj:15]: let's configure versions separately using targets

File: CarSharingApp.PublicApi/appsettings.Production.json [hunk 5]
```diff
@@ -2,10 +2,10 @@
   // Note:
   // Production env was set to test how some of the endpoints work on server side DBs as MongoDbAtlas
 
-  //"ConnectionStrings": {
-  //  "MongoDbAtlasCloud": "", // Azure KeyVault
-  //  "MSSQL": "server connection string"
-  //},
+  "ConnectionStrings": {
+    "MongoDbAtlasCloud": "", // Azure KeyVault
+    "MSSQL": "Server=internship-02;Integrated Security=true;Initial Catalog=CarSharingAppDb;Trusted_Connection=True;TrustServerCertificate=True;"
```
Comment (VadzimPapko) [inline CarSharingApp.PublicApi/appsettings.Production.json:7]: confiture as Azure KeyVault

**Review by VadzimPapko — APPROVED** [review]

File: Application/Services/ActionNotesService.cs (hunk 1 above)
Comment (hlebwritescode) [reply Application/Services/ActionNotesService.cs:24]: Deleted all comments

File: Application/Services/ActionNotesService.cs (hunk 3 above)
Comment (hlebwritescode) [reply Application/Services/ActionNotesService.cs:46]: Fixed

File: Application/Services/ActionNotesService.cs (hunk 2 above)
Comment (hlebwritescode) [reply Application/Services/ActionNotesService.cs:35]: fixed

File: CarSharingApp.PublicApi/CarSharingApp.PublicApi.csproj (hunk 4 above)
Comment (hlebwritescode) [reply CarSharingApp.PublicApi/CarSharingApp.PublicApi.csproj:15]: Added Directory.Build.targets file and using it as a file for updating versions of the same packages into different projects.

**Pull request merged**